example-osgi-library/pom.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project
+	xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>life.qbic</groupId>
+		<artifactId>parent-pom</artifactId>
+		<version>3.1.3</version>
+	</parent>
```

Comment:
Is this already OSGI ready?

Reply:
oh, this was a leftover. I deleted it. Thanks for pointing that out! :)